### PR TITLE
New fragment for positive test result without consent (EXPOSUREAPP-3979)

### PR DESCRIPTION
**Pull request by chiljamgossow**
New fragment for positive test result without consent for submission

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/testresult/SubmissionTestResultNoConsentViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    private val showRedeemedTokenWarning = SingleLiveEvent<Unit>()
+    private var wasRedeemedTokenErrorShown = false
+    private val tokenErrorMutex = Mutex()
+
+    val uiState: LiveData<TestResultUIState> = combineTransform(
+        submissionRepository.deviceUIStateFlow,
+        submissionRepository.testResultReceivedDateFlow
+    ) { deviceUiState, resultDate ->
+
+        tokenErrorMutex.withLock {
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/testresult/SubmissionTestResultNoConsentViewModel.kt:30]: I copy-pasted this. Not sure why it is necessary

File: Corona-Warn-App/src/main/res/drawable/ic_lock.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="16dp"
+    android:height="20dp"
+    android:viewportWidth="16"
+    android:viewportHeight="20">
+  <path
```
Comment (Oliver-Zimmerman) [inline Corona-Warn-App/src/main/res/drawable/ic_lock.xml:6]: Is this icon the same for dark and light theme?

**Review by Oliver-Zimmerman — APPROVED** [review]
LGTM 👍

File: Corona-Warn-App/src/main/res/drawable/ic_lock.xml (hunk 2 above)
Comment (chiljamgossow) [reply Corona-Warn-App/src/main/res/drawable/ic_lock.xml:6]: Yes, all icons are white in both modes

**Review by AlexanderAlferov — APPROVED** [review]

**Pull request merged**